### PR TITLE
CONCF-1032 Fix titleObserver being called when not supposed to

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-1032

/cc @bognix @hakubo

### CODE REVIEW

File: front/scripts/main/components/CuratedContentEditorItemFormComponent.ts [hunk 1]
```diff
@@ -18,11 +18,14 @@ App.CuratedContentEditorItemFormComponent = Em.Component.extend(
 		maxLabelLength: 48,
 		debounceDuration: 250,
 
+		// Force one way binding
+		model: Em.computed.oneWay('attrs.model'),
```
Comment (hakubo) [inline front/scripts/main/components/CuratedContentEditorItemFormComponent.ts]: hmm can we make it default somehow?

Comment (hakubo) [inline front/scripts/main/components/CuratedContentEditorItemFormComponent.ts]: like reopen component class 
and do
`attrs: Em.computed.oneWay('attrs')`

(as a separate PR anyway)

Comment (rogatty) [inline front/scripts/main/components/CuratedContentEditorItemFormComponent.ts]: There is http://discuss.emberjs.com/t/one-way-bounded-components-properties-future-proofing-1-x-to-2-0/7300 but let's leave it for later.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**